### ISSUE TITLE
Why did you leave Docker?

### ISSUE THREAD

**Issue by rovolution** [ChristopherBiscardi/ama#8]
Hi Chris,

Big fan of your work! Keep doing what you do for the JS/open-source community!

I know that you played a large role in developing the UI philosophies and toolchains used at Docker, and I saw that you recently decided to move on.

The company is obviously having a lot of success now, thanks to the hard work of yourself and others. What made you decide that it was time to go?

**Comment by ChristopherBiscardi** [comment]
When I went to work with Docker, there was no Design team, No Frontend team,
etc. There were a limited set of applications, mostly Django with assorted JS
libs based on which page you happened to view. I reset the approach to UI so it
would scale to 7-10+ applications with a small team. During that time we
deployed React, Redux, PostCSS and CSS Modules in Universal, Client and Desktop
configurations across an increasing number of greenfield and legacy projects.

I am extremely proud of the UI team we grew during my tenure at Docker. Today,
this team is capable of driving new initiatives and growing on their own, which
puts a nice cap (for me) on the work I set out to do when I started.

So, simply speaking, I left because my work was done and there is new work I
wish to pursue.
